run_me.sh

```diff
@@ -1,42 +1,46 @@
-cd src/resources || exit
-
 # Create index
-STAR  --runMode genomeGenerate --genomeFastaFiles Mus_musculus.GRCm38.dna_rm.chr19.fa --sjdbGTFfile Mus_musculus.GRCm38.88.chr19.gtf --genomeSAindexNbases 11
+STAR \
```

Comment:
I recommend calling tools in this way, whenever there are multiple arguments and/or long param names or values. It's much easier to read, maintain/modify and review